client/src/Constants.js

```diff
@@ -0,0 +1,10 @@
+/* Store constant values here */
+
+export const minPasswordLength = 6;
+export const minPasswordErrMsg = `Your password must be at least ${minPasswordLength} characters long`;
+export const takenEmailErrMsg = "That email is already taken";
+
+const serverUrl = "http://localhost:5000/";
+const clientUrl = "http://localhost:3000/";
```

Comment:
Instead of doing this, you can set the  proxy field on package.json to be the server url: https://create-react-app.dev/docs/proxying-api-requests-in-development/
This way you won't need to include the base url and any requests for a  route that does not exist on the frontend will be redirected to the server

Reply:
Ok I can do that